### DISCUSSION TITLE
Next page button not working

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#6045] [Help] [answered]
I am using SQL 2008.  The next page button was not working.  See screenshot.
Not sure, something I can change to make the next page button working.

![image](https://user-images.githubusercontent.com/44040606/137751621-30761eab-07a1-4de4-9299-22bafb74e7a5.png)
![image](https://user-images.githubusercontent.com/44040606/137751890-e26c00bd-ea06-46e4-803d-b069380042c8.png)

**Comment by wezmag** [comment]
SQL 2008 does not support FETCH NEXT and OFFSET.
You need to change dialect settings.
There is a wiki page for this.
https://github.com/serenity-is/Serenity/wiki/Troubleshooting#sql-and-connections

**Reply by wezmag** [reply]
If you are using the dotnet core version, you can add `Dialect` in your connectionstring.

For example:
```
"Northwind": {
  "ConnectionString": "Server=(localdb)\\MsSqlLocalDB;Database=Serene_Northwind_v1;Integrated Security=true",
  "ProviderName": "System.Data.SqlClient",
  "Dialect": "SqlServer2008"
}
```

**Comment by ghost** [comment, marked as answer]
This problem was resolved after the database was upgraded  to SQL 2014
